torch/lib/THCUNN/SpatialDepthwiseConvolution.cu

```diff
@@ -11,7 +11,17 @@
 #include "SharedMem.cuh"
 #include "common.h"
 
-template <typename T, typename AccT, typename IndexType>
+
+const int WARP_SIZE = 32;
+const int MAX_BLOCK_SIZE = 256;
```

Comment:
Why clip blocks to 256 threads?

Reply:
In my crude benchmarks, 256 was better than 512 and 1024, but some degree of autotuning/heuristics might improve performance more